src/server/controllers/pokemon.js

```diff
@@ -30,6 +30,7 @@ const getPokeApi = async() => {
 }
 
 const createPokemons = async(req, res) => {
+    const {} = req.body
```

Comment:
Is this supposed to be empty?

Reply:
this was when Steve was talking about authorising the createPokemons fetch, req.body would contain an admin token or password. we dont fetch with a json body yet but when we do the authentication will be in there

Reply:
Ok yes makes sense.

Reply:
Awesome :)